client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -337,6 +337,15 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		.generated-design-picker__previews {
 			min-height: calc( 100vh - 172px );
 			margin-bottom: 84px;
+
+			.web-preview__inner {
+				&.is-selected {
+					opacity: 1;
+					pointer-events: auto;
+					position: relative;
+					z-index: 1;
```

Comment:
[nit] Prefer to make a mixin for it to avoid the duplication and make the connection between here and L507 (they are a little far apart. Besides, it would be great to add a short description to explain why we only show it when previewing on the mobile and link to the related issue.

Reply:
I agree! Nice suggestion 🙂